include/deal.II/grid/reference_cell.h

```diff
@@ -806,6 +806,37 @@ class ReferenceCell
    */
   constexpr ReferenceCell(const std::uint8_t kind);
 
+  /**
+   *
+   */
```

Comment:
?

Reply:
oops, I forgot to fill those out. I'll fix it!